examples/quagga-ixp/topo.py

```diff
@@ -74,4 +72,4 @@ def __init__(self):
                                 nodeConfig=quaggaSvcConfig)
 
             # Attach the quaggaContainer to the IXP Fabric Switch
-            self.addLink(quaggaContainer, ixpfabric)
+            self.addLink(quaggaContainer, ixpfabric, fast=False)
```

Comment:
Thanks for providing an explanation of this change in the PR -- can you add a brief comment to the file as well?